spec/scenario/Flywheel/Grants.scen

```diff
@@ -50,8 +50,7 @@ Macro InitSpeeds
     Mint Coburn 6e18 cBAT--tokenbalance = 6e18 / 2e9 = 3e9
     EnterMarkets Coburn cBAT
     Borrow Coburn 1e18 cZRX
-    Comptroller SetCompSpeed cZRX 1
-    Comptroller SetCompSpeed cBAT 1
+    Comptroller SetCompSpeeds (cZRX cBAT) (1 1) (1 1)
     Comptroller RefreshCompSpeeds
```

Comment:
shouldn't the refresh comp speeds calls be deleted? ( unless you are testing some transition from history ) This uses the old utilization based comp speed calibration, and will only move values away from what they should be , right? https://github.com/compound-finance/compound-protocol/blob/master/tests/Contracts/ComptrollerScenario.sol#L43

Reply:
Thanks, fixed!